xendit_pos/models/xendit_client.py

```diff
@@ -104,3 +104,26 @@ def _get_invoice(self, invoice_id):
             }
 
         return response
+
+    def _cancel_invoice(self, invoice_id):
+        endpoint = self.tpi_server_url + '/payment/xendit/invoice/' + invoice_id + '/expire'
+        headers = self._generate_header(self)
+
+        try:
+            res = requests.post(endpoint, headers=headers, timeout=10)
+        except requests.exceptions.RequestException as err:
+            print ("OOps: ",err)
```

Comment:
does the log still need in production @andykim ? where it will be show?

Reply:
@yanoandri . I updated the error handller. Can you check it again?